numba/tests/test_unsafe_intrinsics.py

```diff
@@ -152,3 +153,63 @@ def use_dump_refcount():
         tupty = types.Tuple.from_types([aryty] * 2)
         self.assertIn(pat.format(aryty), output)
         self.assertIn(pat.format(tupty), output)
+
+
+class TestZeroCounts(TestCase):
+    def test_zero_count(self):
+        lz = njit(lambda x: leading_zeros(x))
+        tz = njit(lambda x: trailing_zeros(x))
+
+        evens = [2, 42, 126, 128]
+
+        for T in types.unsigned_domain:
+            assert tz(T(0)) == lz(T(0)) == T.bitwidth
+            for i in range(T.bitwidth):
+                val = T(2 ** i)
+                assert lz(val) + tz(val) + 1 == T.bitwidth
```

Comment:
Numba uses the `unittest` functions for assertion, i.e. `self.assertEqual(lz(val) + tz(val) + 1, T.bitwidth)`, for consistency please could these tests be adjusted to not use `assert`, thanks.

Reply:
Ah, right. I normally use pytest, and totally forgot about that.